pennylane/tape/tape.py

```diff
@@ -1547,16 +1609,26 @@ def copy(self, copy_operations=False):
             tape._prep = [copy.copy(op) for op in self._prep]
             tape._ops = [copy.copy(op) for op in self._ops]
             tape._measurements = [copy.copy(op) for op in self._measurements]
+            tape._graph = None
```

Comment:
If the operations are copied, the original `_graph` would have outdated references, therefore we can not copy it over.